README.md

```diff
@@ -1,117 +1,179 @@
+Veracity is a command line tool providing support for inspecting the DataTrails native `MERKLE_LOG` verifiable data structures.
 
-Familiarity with a command line environment on your chosen platform is assumed
-by this README.
-
-A general familiarity with verifiable data structures, and in particular binary
-merkle trees, would be advantageous when using `veractity` but is not required.
-
-## Support
-
-We provide pre-built native binaries for linux, mac, and windows. The
-following architectures are supported:
-
-| Platform      | Architecture |
-| :--------     | -----------: |
-| MacOS(darwin) | arm64        |
-| MacOS(darwin) | x86_64       |
-| Linux         | arm64        |
-| Linux         | x86_64       |
-| Windows       | x86_64       |
-| Windows       | i386         |
-
-The linux binaries can also be used in Windows Subsystem for Linux.
+A general familiarity with _verifiable data structures_, and in particular binary merkle trees, would be advantageous, but is not required.
 
 ## Installation
 
+Veracity provides native binaries for [Mac OS](#mac-install), [Linux](#linuxwsl-install) and Windows on the [releases](https://github.com/datatrails/veracity/releases) page.
+
+_Note_: For The Windows Subsystem for Linux (WSL), use the Linux binaries.
 
-Installation is a manual process:
+| OS      | Platform  | Architecture |
+| :------ | :-------- | -----------: |
+| Mac     | `darwin`  | `arm64`      |
+| Mac     | `darwin`  | `x86_64`     |
+| Linux   | `linux`   | `arm64`      |
+| Linux   | `linux`   | `x86_64`     |
+| Windows | `windows` | `x86_64`     |
+| Windows | `windows` | `i386`       |
 
+1. Select the desired release from the [releases page](https://github.com/datatrails/veracity/releases).
 1. Download the archive for your host platform
-2. Extract the archive
-3. Set the file permissions
-4. Move the binary to a location on your PATH
+1. Extract the archive
+1. Set the file permissions
+1. Move the binary to a location on your PATH
 
-For example, for the Linux or Darwin OS the following steps would be conventional
+Or, follow these commands to install the latest build.
 
-```
-PLATFORM=Darwin
+### Mac Install
+
+```console
+PLATFORM=darwin
 ARCH=arm64
+cd $TMPDIR
 curl -sLO https://github.com/datatrails/veracity/releases/latest/download/veracity_${PLATFORM}_${ARCH}.tar.gz
 tar -xf veracity_${PLATFORM}_${ARCH}.tar.gz
 chmod +x ./veracity
-./veracity --help
+mv ./veracity $HOME/.local/bin/
+veracity --help
 ```
 
-Set PLATFORM and ARCH according to you environment. Select the desired release
-from the [releases page](https://github.com/datatrails/veracity/releases) as VERSION (Omitting the 'v').
+### Linux/WSL Install
 
-The last step should report usage information. Usual practice is to move the
-binary into a location on your $PATH. For example:
-
-```
-mkdir -p $HOME/bin
-mv ./veracity $HOME/bin/
-which veracity
+```console
+PLATFORM=linux
```

Comment:
arch and platform can be determined by commands:

     PLATFORM=$(uname -s | tr [:upper:] [:lower:])
     ARCH=$(uname -m)

Reply:
nice!